test/matrix.yml

```diff
@@ -2,6 +2,7 @@ OS:
   - ubuntu-1404
   - ubuntu-1604
   - ubuntu-1804
+  - ubuntu-2004
```

Comment:
https://github.com/elastic/ansible-elasticsearch/blob/master/test/matrix-6x.yml also need to be updated.

Note that we now have 2 different test jobs for 6.x and 7.x despite GitHub only show results from one job (need to fix it soon...):
- 6.x: https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request-6x/20/
- 7.x: https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/250/